docs/development/contributing/ways-to-contribute.rst

```diff
@@ -0,0 +1,95 @@
+* You have checked `existing bug reports <https://github.com/HumanCompatibleAI/imitation/issues?q=is%3Aissue+label%3Abug+is%3Aopen>`_ to make sure that your issue has not already been reported, and
+* You have a minimal, reproducible example of the issue.
+
+When submitting a bug report, please **include the following information**\ :
+
+
+* A clear, concise description of the bug,
+* A minimal, reproducible example of the bug, with installation instructions, code, and error message,
+* Information on your OS name and version, Python version, and other relevant information (e.g. hardware configuration if using the GPU), and
+* Whether the problem arose when upgrading to a certain version of imitation, and if so, what version.
+
+Suggesting new features
+-----------------------
+
+This section explains how you can submit a new feature request, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your request and intended use cases and find related suggestions.
+
+You can submit a new bug report by creating an issue on `GitHub <https://github.com/HumanCompatibleAI/imitation/issues/new>`_ and labeling it as an *enhancement*. **Before you do so, please make sure that**\ :
+
+
+* You have checked the `documentation <https://imitation.readthedocs.io/en/latest/>`_ that relates to your request, as it may be that such feature is already available,
+* You have checked `existing feature requests <https://github.com/HumanCompatibleAI/imitation/issues?q=is%3Aissue+label%3Aenhancement+is%3Aopen+>`_ to make sure that there is no similar request already under discussion, and
+* You have a minimal use case that describes the relevance of the feature.
+
+When you **submit the feature request**:
+
+
+* Use a clear and descriptive title for the GitHub issue to easily identify the suggestion.
+* Describe the current behavior, and explain what behavior you expected to see instead and why.
+* If you want to request an API change, provide examples of how the feature would be used.
+* If you want to request a new algorithm implementation, please provide a link to the relevant paper or publication.
+
+Contributing to the documentation
+---------------------------------
+
+One of the simplest ways to get started contributing imitation is through improving the documentation. Currently, our documentation some gaps, and we would love to have you help us fill them. You can help by adding missing sections of the API docs, editing existing content to make it more readable, clear and accessible, or contribute with new content, such as tutorials and FAQs.
+
+If you have struggled to understand something about our codebase and managed to figure it out in the end, please consider improving the relevant documentation section, adding a tutorial or a FAQ entry, so that other users can learn from your experience.
```

Comment:
```suggestion
If you have struggled to understand something about our codebase and managed to figure it out in the end, please consider improving the relevant documentation section, or adding a tutorial or a FAQ entry, so that other users can learn from your experience.
```